test/upgradability/upgradability.proxy.ts

```diff
@@ -250,6 +255,66 @@ contract("SavingAccount() proxy", async (accounts) => {
             // ======================
             const upgradeBank = await upgrades.upgradeProxy(bankProxy.address, Bank);
         });
+
+        it("SavingAccount - initFINAddress(): from V1.1 to latest", async () => {
+            // ==================
+            // SavingAccount V1.1
+            // ==================
+            const UtilsV1_1 = await ethers.getContractFactory("UtilsV1_1");
+            const utilsV1_1 = await UtilsV1_1.deploy();
+
+            const SavingLibV1_1 = await ethers.getContractFactory("SavingLibV1_1", {
+                libraries: {
+                    UtilsV1_1: utilsV1_1.address,
+                },
+            });
+            const savingLibV1_1 = await SavingLibV1_1.deploy();
+
+            const SavingAccountV1_1 = await ethers.getContractFactory("SavingAccountV1_1", {
+                libraries: { SavingLibV1_1: savingLibV1_1.address, UtilsV1_1: utilsV1_1.address },
+            });
+
+            // ====================
+            // SavingAccount latest
+            // ====================
+            const Utils = await ethers.getContractFactory("Utils");
+            const utils = await Utils.deploy();
+
+            const SavingLib = await ethers.getContractFactory("SavingLib", {
+                libraries: {
+                    Utils: utils.address,
+                },
+            });
+            const savingLib = await SavingLib.deploy();
+
+            const SavingAccount = await ethers.getContractFactory("SavingAccount", {
+                libraries: { SavingLib: savingLib.address, Utils: utils.address },
+            });
+
+            // ========================
+            // SavingAccount V1.1 Proxy
+            // ========================
+            const savingAccountProxy = await upgrades.deployProxy(
+                SavingAccountV1_1,
+                [[], [], DUMMY],
+                { initializer: "initialize", unsafeAllow: ["external-library-linking"] }
+            );
+
+            // ==========================
+            // SavingAccount latest Proxy
+            // ==========================
+            const SAV = await upgrades.upgradeProxy(savingAccountProxy.address, SavingAccount, {
+                unsafeAllow: ["external-library-linking"],
+            });
+
+            // call initFINAddress() & verify FIN address after upgrade
+            await SAV.initFINAddress();
+            const FINAddress = SAV.FIN_ADDR();
+            expect(FINAddress).to.be.equal(FINAddress);
```

Comment:
Variables have the same name `FINAddress`. Needs to be different.

Reply:
renamed it